src/app/authentication/login.component.ts

```diff
@@ -23,8 +20,7 @@ export class LoginComponent implements OnInit {
     header: any;
     authMessages: any;
 
-    constructor(public toastr: ToastsManager, vcr: ViewContainerRef, private _sharedData: SharedDataService, private _activatedRoute: ActivatedRoute, private _langService: LanguageService, private _authService: AuthService, private _router: Router, fb: FormBuilder) {
-        this.toastr.setRootViewContainerRef(vcr);
+    constructor( vcr: ViewContainerRef, private _sharedData: SharedDataService, private _activatedRoute: ActivatedRoute, private _langService: LanguageService, private _authService: AuthService, private _router: Router, fb: FormBuilder) {
```

Comment:
Why is the toastr library's import and reference statements missing in this file?

Reply:
That's the great thing about the new ngx-toastr module. We no more need to pass 'viewContainerRef'

Reply:
Oh I see. Why is the toastr library import removed? Are we not using it here?

Reply:
Yes! it was not being used in the component. Hence, I removed it.